Digipost.Signature.Api.Client.Core/Utilities/LoggingUtility.cs

```diff
@@ -0,0 +1,33 @@
+using System;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using NLog.Extensions.Logging;
+
+namespace Digipost.Signature.Api.Client.Core.Tests.Utilities
+{
+    public static class LoggingUtility
+    {
+        internal static IServiceProvider CreateServiceProviderAndSetUpLogging()
+        {
+            var services = new ServiceCollection();
+            
+            services.AddSingleton<ILoggerFactory, LoggerFactory>();
+            services.AddSingleton(typeof(ILogger<>), typeof(Logger<>));
+            services.AddLogging((builder) => builder.SetMinimumLevel(LogLevel.Trace));
+
+            var serviceProvider = services.BuildServiceProvider();
+            SetUpLoggingFortTesting(serviceProvider);
+
+            return serviceProvider;
+        }
+        
+        private static void SetUpLoggingFortTesting(IServiceProvider serviceProvider)
```

Comment:
```suggestion
        private static void SetUpLoggingForTesting(IServiceProvider serviceProvider)
```

Gjøres med refaktorering og ikke her i Github

Reply:
Kresen fyr altzå.

Reply:
🙈